lib/pure/strutils.nim

```diff
@@ -84,31 +84,42 @@ from std/private/strimpl import cmpIgnoreStyleImpl, cmpIgnoreCaseImpl, startsWit
 
 
 const
-  Whitespace* = {' ', '\t', '\v', '\r', '\l', '\f'}
+  Whitespace* = {' ', '\t', '\v', '\r', '\n', '\f'}
```

Comment:
also, a separate PR would be welcome to change all instances of `\l` to `\n` in nim repo (and it'd only do that plus maybe related changes)